specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Well, indexing (in the database sense of index) and sorting don't necessarily have to be done on the same columns. But yes, that makes sense.